src/cljs/org/broadinstitute/firecloud_ui/page/workspace/data/copy_data_entities.cljs

```diff
@@ -29,8 +29,11 @@
          (when (:server-error @state)
            [:div {:style {:marginTop "0.5em"}}
             [comps/ErrorViewer {:error (:server-error @state)}]])
-         [:div {:style {:marginTop "1em"}}
-          [comps/Button {:text "Copy"
+         [:div {:style {:display "inline-flex" :marginTop "1em"}}
+           [comps/Button {:text "Done"
+                          :style {:marginRight 20}
+                          :onClick (:done props) }]
+           [comps/Button {:text "Copy"
                          :onClick #(let [selected (react/call :get-selected-entities (@refs "EntitySelector"))]
                                      (if (empty? selected)
```

Comment:
We are removing the "Done" button from the OKCancelForm but we still want it to appear in the Copy Entities section when we're importing data from a different workspace